### PR TITLE
fix: upgrade multiple dependencies with Snyk

### PR DESCRIPTION
**Pull request by RonSherfey**
Snyk has created this PR to upgrade:
  - gatsby from 2.31.0 to 2.31.1.
    See this package in npm: 
  - gatsby-image from 2.7.0 to 2.10.0.
    See this package in npm: 
  - gatsby-plugin-lodash from 3.6.0 to 3.9.0.
    See this package in npm: 
  - gatsby-plugin-manifest from 2.8.0 to 2.11.0.
    See this package in npm: 
  - gatsby-plugin-react-helmet from 3.6.0 to 3.9.0.
    See this package in npm: 
  - gatsby-plugin-sharp from 2.10.1 to 2.13.4.
    See this package in npm: 
  - gatsby-plugin-sitemap from 2.8.0 to 2.11.0.
    See this package in npm: 
  - gatsby-plugin-styled-components from 3.6.0 to 3.9.0.
    See this package in npm: 
  - gatsby-remark-autolink-headers from 2.7.0 to 2.10.0.
    See this package in npm: 
  - gatsby-remark-copy-linked-files from 2.6.0 to 2.9.0.
    See this package in npm: 
  - gatsby-remark-images from 3.7.0 to 3.10.0.
    See this package in npm: 
  - gatsby-source-filesystem from 2.7.0 to 2.10.0.
    See this package in npm: 
  - gatsby-transformer-csv from 2.6.0 to 2.9.0.
    See this package in npm: 
  - gatsby-transformer-remark from 2.12.0 to 2.15.0.
    See this package in npm: 
  - gatsby-transformer-sharp from 2.8.0 to 2.11.0.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/ronsherfey/project/9ae43a7a-6064-44cd-9636-e4924f663010?utm_source=github&utm_medium=upgrade-pr

### CODE REVIEW

**Comment by samajammin** [comment]
Thanks for the PR @RonSherfey!

@wackerow assigning to you - could you please run & build this locally & spot check the app functions as expected?

**Review by wackerow — APPROVED** [review]
@samajammin LGTM... build went find, went through all major sections and thing seem to be working as expected. 👍

File: package.json [hunk 1]
```diff
@@ -19,27 +19,25 @@
...
+    "gatsby-remark-copy-linked-files": "^2.9.0",
+    "gatsby-remark-images": "^3.10.0",
+    "gatsby-source-filesystem": "^2.10.0",
     "gatsby-source-graphql": "^2.13.0",
-    "gatsby-transformer-csv": "^2.3.10",
+    "gatsby-transformer-csv": "^2.9.0",
     "gatsby-transformer-gitinfo": "^1.1.0",
-    "gatsby-transformer-remark": "^2.8.16",
-    "gatsby-transformer-sharp": "^2.5.5",
     "lodash": "^4.17.21",
```
Comment (samajammin) [inline package.json:41]:
```suggestion
    "gatsby-transformer-sharp": "^2.11.0",
    "lodash": "^4.17.21",
```

**Comment by wackerow** [comment]
@all-contributors please add @RonSherfey for code

**Pull request merged**